cli/command/system/prune.go

```diff
@@ -34,7 +34,7 @@ func NewPruneCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags := cmd.Flags()
 	flags.BoolVarP(&opts.force, "force", "f", false, "Do not prompt for confirmation")
 	flags.BoolVarP(&opts.all, "all", "a", false, "Remove all unused images not just dangling ones")
-	flags.Var(&opts.filter, "filter", "Provide filter values (e.g. 'until=<timestamp>')")
+	flags.Var(&opts.filter, "filter", "Provide filter values (e.g. 'label=<key>=<value>')")
```

Comment:
This should have an annotation (like in [cli/command/stack/deploy.go#L46](https://github.com/docker/cli/blob/37ccc00d0e14461bcf29e98669773625dfed2fce/cli/command/stack/deploy.go#L46))

```go
flags.SetAnnotation("filter", "version", []string{"1.28"})
```

Same for the other `prune --filter` flags